src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -2858,9 +2910,12 @@ def __init__(self, config):
     @add_start_docstrings_to_model_forward(BIG_BIRD_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @add_code_sample_docstrings(
         processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
+        checkpoint="vumichien/token-classification-bigbird-roberta-base",
```

Comment:
Wow, you have trained a token classification model ..? 💯

Reply:
Sorry, it's just a random weight...(but maybe I will try to train bigbird for token classification in the near future 😅). The reason why I didn't use a model from `hf-internal-testing` (hf-internal-testing/tiny-random-bigbird_pegasus) is that I think it's also the random weight model but the output is too long. But if you think it's not a good way, I will revise with this checkpoint `hf-internal-testing/tiny-random-bigbird_pegasus`

Reply:
It's OK, it is completely fine.

However, if `vumichien/token-classification-bigbird-roberta-base` has random weights, it is a good idea to have the name like

```
vumichien/token-classification-bigbird-roberta-base-random
```

This way, the doc reader and hub users won't be confused 😄

Reply:
Ah, I see. I will change the checkpoint name